po/fr.po

```diff
@@ -15041,331 +15021,305 @@ msgstr "Installer des paquets source"
 
 #: /tmp/fish/implicit/share/completions/dpkg.fish:9
-#, fuzzy
 msgid "List contents of .deb"
-msgstr "Évaluer le contenu d'un fichier"
+msgstr "Lister le contenu du .deb"
 
 #: /tmp/fish/implicit/share/completions/dpkg.fish:10
-#, fuzzy
 msgid "Show .deb information"
-msgstr "Obtention des informations utilisateur impossible."
+msgstr "Obtenir les informations du .deb"
 
 #: /tmp/fish/implicit/share/completions/dpkg.fish:11
-#, fuzzy
 msgid "List packages matching pattern"
-msgstr "Lister le contenu d'un paquet correspondant au motif"
+msgstr "Lister les paquets correspondant au motif"
 
 #: /tmp/fish/implicit/share/completions/dpkg.fish:12
-#, fuzzy
 msgid "List contents of packages"
-msgstr "Lister le contenu d'un paquet correspondant au motif"
+msgstr "Lister le contenu des paquets"
 
 #: /tmp/fish/implicit/share/completions/dpkg.fish:13
-#, fuzzy
 msgid "Print status of package"
-msgstr "Actualiser la liste des paquets"
+msgstr "Afficher le statut du paquet"
 
 #: /tmp/fish/implicit/share/completions/dpkg.fish:14
-#, fuzzy
 msgid "Search for packages containing file"
-msgstr "Cherche les paquet contenant le motif"
+msgstr "Chercher les paquets contenant le fichier"
 
 #: /tmp/fish/implicit/share/completions/dpkg-reconfigure.fish:3
-#, fuzzy
 msgid "Set configuration frontend"
-msgstr "Spécifier un fichier de configuration"
+msgstr "Définir l’interface de configuration"
 
 #: /tmp/fish/implicit/share/completions/dpkg-reconfigure.fish:4
 msgid "Set priority threshold"
-msgstr ""
+msgstr "Définir le seuil de priorité"
 
 #: /tmp/fish/implicit/share/completions/dpkg-reconfigure.fish:5
 msgid "Use current default ("
-msgstr ""
+msgstr "Utiliser le réglage par défaut ("
 
 #: /tmp/fish/implicit/share/completions/dpkg-reconfigure.fish:6
+#, fuzzy
 msgid "Show only unseen question"
-msgstr ""
+msgstr "N’afficher que les questions pour novices"
 
 #: /tmp/fish/implicit/share/completions/dpkg-reconfigure.fish:7
-#, fuzzy
 msgid "Reconfigure also inconsistent packages"
-msgstr "Supprimer des paquets source"
+msgstr "Également reconfigurer les paquets incohérents"
 
 #: /tmp/fish/implicit/share/completions/dpkg-reconfigure.fish:8
+#, fuzzy
 msgid "Prevent reloading templates"
-msgstr ""
+msgstr "Empêcher le rechargement des patrons"
 
 #: /tmp/fish/implicit/share/completions/dropbox.fish:1
-#, fuzzy
 msgid "Get current status of the dropboxd"
-msgstr "Inverser l'état de sortie de la tâche"
+msgstr "Obtenir l’état actuel de dropboxd"
 
 #: /tmp/fish/implicit/share/completions/dropbox.fish:2
-#, fuzzy
 msgid "Provide help"
-msgstr "Afficher les URIs"
+msgstr "Fournir de l’aide"
 
 #: /tmp/fish/implicit/share/completions/dropbox.fish:3
 msgid "Return 1 if dropbox is running"
-msgstr ""
+msgstr "Répondre 1 si dropbox est en fonctionnement"
 
 #: /tmp/fish/implicit/share/completions/dropbox.fish:4
 msgid "Ubuntu: automatically start dropbox at login"
-msgstr ""
+msgstr "Ubuntu : lancer automatiquement dropbox à la connexion"
```

Comment:
Consistency / typography: albeit typographically correct, the space preceding the colon is not present in other translations, which seems wise considering it needs to be non-breaking, a feature not supported by straight text output (AFAIK).